aws/scenarios/vm/os/unix.go

```diff
@@ -0,0 +1,22 @@
+package os
+
+type unix struct{}
+
+func (unix) Visit(v Visitor)                     { v.VisitUnix() }
+func (unix) GetAMIArch(arch Architecture) string { return string(arch) }
+func (unix) GetDefaultInstanceType(arch Architecture) string {
+	return getDefaultInstanceType(arch)
+}
+func (unix) GetTenancy() string    { return "default" }
+func (unix) GetConfigPath() string { return "/etc/datadog-agent/datadog.yaml" }
+
+func getDefaultInstanceType(arch Architecture) string {
+	switch arch {
+	case AMD64Arch:
+		return "t3.large"
```

Comment:
Variables are available for that already

Reply:
What do you mean?

Reply:
Same as for `keyPair` we have a `ConfigMap` parameter for that:
```
	ddinfraDefaultInstanceTypeParamName    = "aws/defaultInstanceType"
	ddinfraDefaultARMInstanceTypeParamName = "aws/defaultARMInstanceType"
```

Reply:
Code updated. Note: I have to export some constants defined in `environment.go` in order to use them in `datadog-agent` repo to set the config map.